bigbluebutton-html5/imports/api/log-client/server/methods/logClient.js

```diff
@@ -1,19 +1,19 @@
 import Logger from '/imports/startup/server/logger';
 import Users from '/imports/api/users';
 
-const logClient = function (type, log, ...args) {
+const logClient = function (type, log, fullInfo) {
   const SERVER_CONN_ID = this.connection.id;
   const User = Users.findOne({ connectionId: SERVER_CONN_ID });
-  const logContents = { ...args };
+  const logContents = { fullInfo };
 
   if (User) {
-    if (User.meetingId === args[0].meetingId) {
-      args[0].validUser = 'vaild';
+    if (User.meetingId === fullInfo.meetingId) {
```

Comment:
I keep getting my user as `invalid` - debugging a bit shows `fullInfo.meetingId` as undefined but `fullInfo.credentials.meetingId` as the right one. Please double check